sdk/communication/azure-communication-callingserver/azure/communication/callingserver/_callingserver_client.py

```diff
@@ -368,4 +387,127 @@ def cancel_participant_media_operation(
         return self._server_call_client.cancel_participant_media_operation(
             cancel_participant_media_operation_request=cancel_participant_media_operation_request,
             **kwargs
-            )
\ No newline at end of file
+            )
+
+    @distributed_trace()
+    def start_recording(
+        self,
+        call_locator,  # type: CallLocator
+        recording_state_callback_uri,  # type: str
+        **kwargs  # type: Any
+    ):  # type: (...) -> StartCallRecordingResult
+
+        if not call_locator:
+            raise ValueError("call_locator cannot be None")
+        if not CallingServerUtils.is_valid_url(recording_state_callback_uri):
+            raise ValueError("recording_state_callback_uri is invalid")
+
+        start_call_recording_request = StartCallRecordingWithCallLocatorRequestConverter.convert(
```

Comment:
@zihzhan-msft

Reply:
I am ok with removing converter.
Some api take *Options parameters which make the api method body looks much complicated, trying to reduce line in client class , and this can also be used in async cleint method.

Reply:
Removed convertor